src/renderer/components/LandingPage/HeaderView.vue

```diff
@@ -133,12 +133,12 @@ export default {
       }
     }
     .faExternal {
-      color: $white-color;
+      color: $pitch;
```

Comment:
Have you double checked this? Changing from white to dark is a bit risky.

Reply:
@naluinui do you agree that the white color for this link will be not really readable?
<img width="194" alt="Untitled" src="https://github.com/rfcx/arbimon-uploader/assets/31901584/aae36fa9-c4a3-4913-9cf6-45aede049bfe">
<img width="64" alt="Untitled2" src="https://github.com/rfcx/arbimon-uploader/assets/31901584/a3831bb5-1103-4782-a2a1-5938027b427d">